resource/wrapper/retryresource/simple_resource.go

```diff
@@ -12,13 +12,19 @@ import (
 	"github.com/giantswarm/operatorkit/resource"
 )
 
-type resourceWrapper struct {
+type simpleResourceConfig struct {
```

Comment:
I prefer `basicResource` over simple but that's nit.

Reply:
sorted